### ISSUE TITLE
Vulkan Portability Layer not loading only when exporting/archiving SDL App via Apple iOS Deployment

### ISSUE THREAD

**Issue by teafella** [libsdl-org/SDL#8307]
I am having a strange issue where I have an SDL + Vulkan iOS App that used to work and exported just fine via Test Flight this past June ( I showed this exported app at a trade show ) and recently when I tried exporting again I am now experiencing a SDL_Init error when using the usual SDL_WINDOW_VULKAN flag when creating window its saying "Failed to load Vulkan Portability library".

Building directly to all iOS sdk supported platforms works just fine (Phone/Pad/Desktop) with the iOS SDL frameworks and MoltenVK.xcframework. I have recently re-created my whole build system from the main branch SDL iOS Demos and tried to add Vulkan support to it all over again but I am still getting this crash only when exported via apple's deployment to any platform. 

This is thoroughly perplexing as I have had this exact app working fine and deploying on AppStore earlier this year and I have rolled back my repository to this exact state from earlier in the year and I am getting this new issue. Could anyone point me in the correct direction to be able to debug this issue? 

Advance thanks to anyone for any help/suggestions!

**Comment by jmatzen** [comment]
I'm also getting this.  Did you figure it out?

**Comment by slouken** [comment]
@danginsburg, do you have any insight here?

**Comment by natbro** [comment]
heya @teafella - just heard about you having this issue, this seems off. Can you share (either here, to nathaniel_brown@apple.com, or through https://feedbackassistant.apple.com) the application id and if possible the IPA you were trying to submit, also if you're still experiencing this, adding me to your TestFlight (nathaniel_brown@apple.com) so I can see what's going on? to the best of my knowledge there should be no issue loading .dylib and frameworks.

**Comment by danginsburg** [comment]
> @danginsburg, do you have any insight here?

No, perhaps @richard-lunarg or @charles-lunarg might be able to provide info.

**Comment by richard-lunarg** [comment]
Are you shipping any layers with the app?

**Comment by teafella** [comment]
Oh hey, thanks for you replies everyone! Last year when I posted I had just given up and rewrote+launched [my app](https://apps.apple.com/us/app/sleepy-vidos/id6448588001) with native windowing haha. 

I just tried to re-export the same exact project in xcode (literally untouched since I made the post above) and it seems to be working just fine now! So genuinely no clue what was happening here... The error was being shown by giving SDL_GetError to SDL_ShowSimpleMessageBox.

Thanks for bringing this issue to my attention again, I will definitely be rolling back to SDL asap since whatever black magic yall did here is actually a lot better than the native windowing ( "Made for iPad" on macOS can be any size like a proper desktop app instead of the annoying preset sizes). Thanks again for all your team's hard work on SDL and Vulkan.
